src/convert_torch_to_tf.py

```diff
@@ -0,0 +1,85 @@
+# __author__ = "Vasudev Gupta"
+# __author_email__ = "7vasudevgupta@gmail.com"
+
+import numpy as np
+import tensorflow as tf
+from tqdm.auto import tqdm
+
+import transformers
+from wav2vec2 import Wav2Vec2Config, Wav2Vec2ForCTC
+
+PREFIX = "wav2vec-ctc/"
+SUFFIX = ":0"
+MAPPING = (
+    ("layer_norm.weight", "layer_norm/gamma"),
+    ("layer_norm.bias", "layer_norm.beta"),
+    ("weight", "kernel"),
+    (".", "/"),
+)
+
+KEYS_TO_IGNORE = [
+    "wav2vec2.masked_spec_embed",
+]
+
+SPECIAL_MAPPING = {
+    "wav2vec2.encoder.pos_conv_embed.conv.weight_g": "wav2vec-ctc/wav2vec2/encoder/pos_conv_embed/conv/weight_g:0",
+    "wav2vec2.encoder.pos_conv_embed.conv.weight_v": "wav2vec-ctc/wav2vec2/encoder/pos_conv_embed/conv/weight_v:0",
+}
```

Comment:
Would it make sense to maintain a separate `config.py` or `config.yaml` to put all of these constants?

Reply:
These are getting used only in this script & very unlikely to change with other checkpoints. So please confirm, If I still need to make separate `config file` for these constants???

Reply:
Okay, if these are specific to this script I think it's fine to keep them here.